.github/workflows/ci.yml

```diff
@@ -64,5 +64,19 @@ jobs:
             echo "Test expectations are out-of-date on the target branch."
             echo "You can run `./tools/maint/rebaseline_tests.py --new-branch`"
             echo "and use it to create a seperate PR."
+            echo "-- This failure is only a warning and can be ignored"
             exit 1
           fi
+      - name: Check test expectations on PR branch
+        run: |
+          git checkout ${{ github.base_name }}
```

Comment:
what is gitub.base_name? I don't see it on https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs

Reply:
Oops, should be base_ref